Brocfile.js

```diff
@@ -17,4 +18,13 @@ var app = new EmberApp();
 // please specify an object with the list of modules as keys
 // along with the exports of each module as its value.
 
-module.exports = app.toTree();
+// Foundation
+app.import('bower_components/foundation/js/foundation.min.js');
```

Comment:
Wouldn't it be better to use [broccoli-static-compiler](http://www.ember-cli.com/#using-broccoli-static-compiler) to collect all the FontAwesome assets?